react/orders101/src/features/BasketOpen.jsx

```diff
@@ -0,0 +1,16 @@
+function BasketOpen({ openCart }) {
+  return (
+    <button onClick={() => openCart()} class="nav__icons" id="cart-icon">
+      <div class="nav__icons__item container">
+        <img
+          class="searchicons"
```

Comment:
Small thing here - it looks like the `searchicons` class is not defined in the css that's being used for the React app. 

I would also probably consider changing the name of this class (since it's plural and doesn't follow the naming convention using kebab-case), but perhaps we can just remove it?

Reply:
searchicons removed